### ISSUE TITLE
Reactify the content analysis

### ISSUE THREAD

**Issue by IreneStr** [Yoast/yoast-components#193]
This is the overview issue for the Reactification of content analysis 
(Issue in Roadmap Repo: https://github.com/Yoast/roadmap/issues/4)

Components:
- [x] Bullet #189 
- [x] IconButton #190 
- [x] AnalysisResult #191 
- [x] CollapsibleHeader #192 
- [x] ContentAnalysis #194

Redux:
- [x] Root reducer https://github.com/Yoast/wordpress-seo/issues/8116
- [x] Content analysis reducer + actions #202

Implementation in WP:
- [x] Store: https://github.com/Yoast/wordpress-seo-premium/pull/1409
- [x] Implement content analysis component:https://github.com/Yoast/wordpress-seo/issues/8116

Mockup:
<img width="644" alt="seo_analysis_reactified_4_psd___100___layer_2__rgb_8__" src="https://user-images.githubusercontent.com/6073772/29068441-27acf922-7c37-11e7-96d2-3771e9d94f0a.png">
Other categories: `Errors` and `Considerations`

**Comment by hedgefield** [comment]
There's been a slight change. This will be step 1 of reactifying the whole Yoast SEO metabox, but in this issue we are _just_ implementing the (collapsible) headers in the analysis results (both for the readability and focus kw tabs). So, this:

![analysis react 1](https://user-images.githubusercontent.com/6073772/28933351-0d28691e-787d-11e7-844e-af98d95bb889.png)

That means **don't** work on the stuff around it (the changed headings and the removal of the extra paper that can be seen in the original mockup). @IreneStr can you see that the relevant issues are put on hold/changed?

For context: after we've built this, we will:

- reactify the snippet preview
- then we add collapsibleheaders to all the papers in these tabs
- then we turn the tabs themselves into collapsible headers (so adding keywords will happen vertically too)
- and _then_ we turn the menu on the side into a tab navigation above the metabox. Eventually, it will look like this:

![accordeon4](https://user-images.githubusercontent.com/6073772/28933504-901f67d2-787d-11e7-9103-b79aaaae1849.png)

But first, headers in the analysis results ;)

**Comment by IreneStr** [comment]
@hedgefield Thanks for the update! I've updated all related issues. 

On a totally different note: `problems` - `improvements` - `good` doesn't sound like a logical sequence to me. 'Problems' and 'improvements' are nouns (and '4 problems' and '2 improvements' make sense), while 'good' is an adjective (and '2 good' sounds weird). However, I unfortunately can't come up with a better name at this moment 😅

**Comment by hedgefield** [comment]
@IreneStr I had the exact same problem 😄  It's the fact that `Problems` and `Improvements` are actionable, there are plenty of words for them, but when you have done something well, it's just kind of a pat on the back bit of feedback, `Good` seems like the only choice there... even when altering the intent of that subheading, something like `Completed` or `Done`, it's still the same problem. The only kind of words that match are in the area of `Successes` or `Wins` or something, but those are very hard to parse, meaning-wise.

**Comment by IreneStr** [comment]
@hedgefield Like I wrote, it's hard to find better wording. `Successes` and `Wins` don't sound right to me as well. Maybe we can ask Marieke when she's back?

**Comment by hedgefield** [comment]
That's a good idea.

**Comment by hedgefield** [comment]
We should remember to also fix this during the reactification https://github.com/Yoast/wordpress-seo/issues/7383

**Comment by IreneStr** [comment]
@hedgefield Any new thoughts about the header names? Marieke is on vacation again 🙃

**Comment by hedgefield** [comment]
@IreneStr Discussed with @moorscode and @omarreiss: Problems - Improvements - Good is fine ;)

**Comment by IreneStr** [comment]
@hedgefield Thanks!

**Comment by IreneStr** [comment]
@hedgefield Yoast/wordpress-seo#7383 is not a button-issue, but a bug in the marking algorithm.

**Comment by hedgefield** [comment]
@IreneStr Okay. So what's next? Is it/can it be fixed? Are you posting the details there?

**Comment by IreneStr** [comment]
Discussed headings with Marieke & Michiel: `Errors` for the grey bullets when it’s broken, and `Considerations` for the other grey bullets (focus keyword 10+ words, one or more stop words in the focus keyword)

\cc @hedgefield

**Comment by IreneStr** [comment]
Fixed by Yoast/wordpress-seo#8116
